service/src/main/kotlin/no/nav/su/se/bakover/service/statistikk/mappers/behandling/BehandlingResultat.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.su.se.bakover.service.statistikk.mappers.behandling
+
+/**
+ * Alle behandlinger som er ferdigbehandlet av førsteinstansen bør ha et resultat.
+ * Må sees i kombinasjon med [BehandlingStatus]
+ */
+enum class BehandlingResultat(val beskrivelse: String) {
+    Innvilget("Behandlingen har blitt innvilget. Dette gjelder søknadsbehandling, revurdering og regulering."),
+    Avslått("Gjelder kun søknadsbehandling: I henhold til lov om supplerende stønad blir en søknad avslått. Tilsvarende resultat for revurdering er opphørt."),
+    Opphørt("En revurdering kan opphøre, mens en søknadsbehandling kan bli avslått."),
+    Uendret("Utfaset eller på vei til å bli utfaset. Dette gjelder stønadsendringer som har mellom 0 og 10% endring og ikke fører til nye oppdragslinjer."),
+    Stanset("Stønadsendring som fører til stans av utbetalingen(e) for en gitt måned og framover. Det motsatte av resultatet [Gjenopptatt]."),
+    Gjenopptatt("Stønadsendring som fører til gjenopptak av utbetaling(e) for en gitt måned og framover. Det motsatte av resultatet [Stanset]."),
+    Opprettholdt("Kun brukt i klagebehandling ved oversendelse til klageinstansen."),
```

Comment:
Hadde det vært like greit å prefikse/postfikse for de tingene som eksplisitt kun kan kjelde for enkelte ting? F.eks KLAGE_OPPRETTHOLDT el.?

Reply:
La på value og postfikset.